stubs/python/build.gradle

```diff
@@ -97,7 +114,9 @@ afterEvaluate {
         dependsOn buildPythonPackage
 
         executable 'bash'
-        args '-c', condaCommand('twine upload dist/*')
+        args '-c', condaCommand("twine upload -u $pypi_user -p $pypi_password dist/*")
```

Comment:
I think properties need to come from `rootProject` to be overridable from command line (we may want to do that for whatever reason).

Also let's check the properties

```groovy
def user = rootProject.findProperty('pypiUser')
if (!user) {
  throw
}
```

etc.

Reply:
For this case I realized we can't throw an exception here because it runs during the config phase.

I'll try making this a generic task that calls exec instead.